client/pages/form-pages/facility-info.tsx

```diff
@@ -0,0 +1,86 @@
+import { Form, Formik } from 'formik';
+import { Button, Field, Radio, Select, Option, CheckboxArray, Textarea } from '@components';
+import { FacilityType, FacilityUsage, StatusPerTransport } from '../../constants';
+
+const initialValues = {
+  facilityType: '',
+  statusCanReg: '',
+  acapEligibilty: '',
+  acapApplication: '',
+  textAreaNoAppAcap: '',
+  facilityUsage: '',
+};
+
+export const FacilityInfo: React.FC = () => {
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      {({ values }: any) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center drop-shadow-sm'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcDarkBlue'>Facility Information</h1>
+              <h3 className=''>Provide detail contact information for your application.</h3>
+            </div>
+
+            <div className='flex flex- w-full justify-start'>
+              <CheckboxArray
+                title='Facility Type'
+                name='facilityType'
+                legend='Please select all that apply'
+                options={FacilityType}
+              />
+            </div>
+
+            <div className='flex flex-1 w-full justify-start'>
+              <Radio
+                title='Status per transport Canada regulations'
+                legend='Select one'
+                name='statusCanReg'
+                horizontal={true}
+                options={StatusPerTransport}
+              ></Radio>
+            </div>
+
+            <div className='flex flex-col w-full px-8 py-4 bg-slate-200'>
+              <Radio
+                legend='Is the project eligible for the federal Airports Capital Assistance Program (ACAP)?'
+                name='acapEligibilty'
+                horizontal={true}
+                options={[
+                  { label: 'Yes', value: 'yes' },
+                  { label: 'No', value: 'no' },
+                ]}
+              />
+
+              <Radio
```

Comment:
All these should be conditional.

Eg:

{values.acapEligibilty == 'yes' && <Radio ....}